trappy/ftrace.py

```diff
@@ -491,6 +491,19 @@ def __init__(self, path=".", name="", normalize_time=True, scope="all",
         self.__populate_metadata()
         self._do_parse()
 
+    def __warn_about_txt_trace_files(self, trace_dat, raw_txt, formatted_txt):
+        self.__get_raw_event_list()
```

Comment:
What about moving the format string at the bottom of the source where we can better format it.
Something like:

```python
def __warn_about_txt_trace_files(self, trace_dat, raw_txt, formatted_txt):
    #...
    TXT_TRACE_WARN_FMT.format(trace_dat,
                              ', '.join(raw_event),
                              raw_txt, formatted_txt)
    #...

# At end of file:
TXT_TRACE_WARN_FMT = """
You appear to be parsing both raw and formatted trace files.

TRAPpy now uses a unified format, if you have the {} file, remove ALL the .txt
files and try again.

If not, you can manually move lines with these events:
   {}
from {} to {}.
"""
```